config/initializers/cors.rb

```diff
@@ -5,12 +5,12 @@
 
 # Read more: https://github.com/cyu/rack-cors
 
-# Rails.application.config.middleware.insert_before 0, Rack::Cors do
-#   allow do
-#     origins "example.com"
-#
-#     resource "*",
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "example.com"
```

Comment:
- [ Required ] As requested by the previous reviewer I think it would be nice if you fix the `Cors` error that occurs in your App when the front-end makes a request to the back-end API. The previous reviewer had suggested that you change the code on line `10` of this file to `origins "*"` this will in turn enable your App to accept HTTP requests from your front-end application thus solving the `Cors` error. I would like to suggest that you refactor your code to what is shown below in order to fix this, please also notice the change on line ten, it's very important.

## Change the code in this file to
```ruby
Rails.application.config.middleware.insert_before 0, Rack::Cors do
  allow do
    origins "*"

    resource "*",
      headers: :any,
      methods: [:get, :post, :put, :patch, :delete, :options, :head]
  end
end
```

## Cors error in project
![Screenshot from 2023-10-22 02-31-01](https://github.com/Odongo006/Hello-React-Front-end/assets/16392046/987dadf9-053d-4e39-9772-eb3ef8d1b64b)